src/static/js/scroll.js

```diff
@@ -0,0 +1,351 @@
+/*
```

Comment:
AWESOME! Much better having all the code related to scrolling on a separated file 🎉